com.unity.render-pipelines.universal/Shaders/Terrain/TerrainDetailLit.shader

```diff
@@ -114,12 +109,10 @@ Shader "Hidden/TerrainEngine/Details/UniversalPipeline/Vertexlit"
             {
                 UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
 
-                half3 bakedGI = SampleLightmap(input.LightmapUV, half3(0.0, 1.0, 0.0));
-
```

Comment:
How are we getting ambient skylight / SH?

Reply:
I ended up bringing the code back here https://github.com/Unity-Technologies/Graphics/pull/3386/commits/95e5f15d6fcf9e1578fb4beb4b329ff109ab92b1 but in a more generic shape so SH sampling is possible for terrain details.